_config.yml

```diff
@@ -24,30 +24,30 @@ timezone: America/New_York
 
 navigation:
   primary:
-    - href: index.html
+    - href: /
       title: Home
-    - href: request.html
+    - href: /request/
       title: Request
-    - href: data.html
+    - href: /data.html
```

Comment:
yeah, exactly. It's a mater of convention but I feel like all of our URLs should start with `/` so it's clear the URL is a full path from the domain (rather than having `baseurl` containing the ending `/` and the URLs not start with `/`).